programs/fundamentals/cp.scl

```diff
@@ -0,0 +1,116 @@
+__map={'='->[], 'A'->[0, 0, 0, 0, 0, 0], 'B'->[0, 0, 0, 0, 0, 1], 'C'->[0, 0, 0, 0, 1, 0], 'D'->[0, 0, 0, 0, 1, 1], 'E'->[0, 0, 0, 1, 0, 0], 'F'->[0, 0, 0, 1, 0, 1], 'G'->[0, 0, 0, 1, 1, 0], 'H'->[0, 0, 0, 1, 1, 1], 'I'->[0, 0, 1, 0, 0, 0], 'J'->[0, 0, 1, 0, 0, 1], 'K'->[0, 0, 1, 0, 1, 0], 'L'->[0, 0, 1, 0, 1, 1], 'M'->[0, 0, 1, 1, 0, 0], 'N'->[0, 0, 1, 1, 0, 1], 'O'->[0, 0, 1, 1, 1, 0], 'P'->[0, 0, 1, 1, 1, 1], 'Q'->[0, 1, 0, 0, 0, 0], 'R'->[0, 1, 0, 0, 0, 1], 'S'->[0, 1, 0, 0, 1, 0], 'T'->[0, 1, 0, 0, 1, 1], 'U'->[0, 1, 0, 1, 0, 0], 'V'->[0, 1, 0, 1, 0, 1], 'W'->[0, 1, 0, 1, 1, 0], 'X'->[0, 1, 0, 1, 1, 1], 'Y'->[0, 1, 1, 0, 0, 0], 'Z'->[0, 1, 1, 0, 0, 1], 'a'->[0, 1, 1, 0, 1, 0], 'b'->[0, 1, 1, 0, 1, 1], 'c'->[0, 1, 1, 1, 0, 0], 'd'->[0, 1, 1, 1, 0, 1], 'e'->[0, 1, 1, 1, 1, 0], 'f'->[0, 1, 1, 1, 1, 1], 'g'->[1, 0, 0, 0, 0, 0], 'h'->[1, 0, 0, 0, 0, 1], 'i'->[1, 0, 0, 0, 1, 0], 'j'->[1, 0, 0, 0, 1, 1], 'k'->[1, 0, 0, 1, 0, 0], 'l'->[1, 0, 0, 1, 0, 1], 'm'->[1, 0, 0, 1, 1, 0], 'n'->[1, 0, 0, 1, 1, 1], 'o'->[1, 0, 1, 0, 0, 0], 'p'->[1, 0, 1, 0, 0, 1], 'q'->[1, 0, 1, 0, 1, 0], 'r'->[1, 0, 1, 0, 1, 1], 's'->[1, 0, 1, 1, 0, 0], 't'->[1, 0, 1, 1, 0, 1], 'u'->[1, 0, 1, 1, 1, 0], 'v'->[1, 0, 1, 1, 1, 1], 'w'->[1, 1, 0, 0, 0, 0], 'x'->[1, 1, 0, 0, 0, 1], 'y'->[1, 1, 0, 0, 1, 0], 'z'->[1, 1, 0, 0, 1, 1], '0'->[1, 1, 0, 1, 0, 0], '1'->[1, 1, 0, 1, 0, 1], '2'->[1, 1, 0, 1, 1, 0], '3'->[1, 1, 0, 1, 1, 1], '4'->[1, 1, 1, 0, 0, 0], '5'->[1, 1, 1, 0, 0, 1], '6'->[1, 1, 1, 0, 1, 0], '7'->[1, 1, 1, 0, 1, 1], '8'->[1, 1, 1, 1, 0, 0], '9'->[1, 1, 1, 1, 0, 1], '+'->[1, 1, 1, 1, 1, 0], '/'->[1, 1, 1, 1, 1, 1]};
```

Comment:
Do you think it's possible to find a more descriptive name for this? Or if not like a comment explaining what it is? Cos new programmers may not understand it.